### PR TITLE
Milestone 1: Add Person, Student and Teacher class.

### PR DESCRIPTION
**Pull request by Ibizugbe**
In this PR, I have completed the current milestone of the project and implemented the required features.
- no linter errors.
- the project is professionally documented.
- the right Gitflow method has been used.
- A person class has been created with instance variables  `@id`, `@name,` and `@age`, with constructors, getters and setters, and public and private methods.
- A student class has been created which depends on the person class.
- A teacher class has been created that depends on the person class.

### CODE REVIEW

File: student.rb [hunk 1]
```diff
@@ -0,0 +1,12 @@
+require './person'
+
+class Student < Person
+  def initialize(classroom)
```
Comment (Munsa1) [inline student.rb]: - [ ] Its a requirement that the constructor extends parent's constructor by adding @classroom and a parameter for it.
e.g
image

File: teacher.rb [hunk 2]
```diff
@@ -0,0 +1,12 @@
+require './person'
+
+class Teacher < Person
+  def initialize(specialization)
```
Comment (Munsa1) [inline teacher.rb]: - [ ] Its a requirement that the constructor extends parent's constructor by adding @Specialization and a parameter for it
e.g
image

**Review by Munsa1 — CHANGES_REQUESTED** [review]
Hi @Ibizugbe, 

Good job so far!
There are some issues that you still need to work on to go to the next project but you are almost there!
<FEEL FREE TO CHANGE THIS MESSAGE TO YOUR PERSONAL WORDS>

## Highlights
:heavy_check_mark: Descriptive Readme
:heavy_check_mark: Passing Linters
:heavy_check_mark: Correct Gitflow
## Required Changes ♻️
_Check the comments under the review._

## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._

You can also consider:

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

File: teacher.rb (hunk 2 above)
Comment (Ibizugbe) [reply teacher.rb]: Thanks a lot, @Munsa1  for pointing out these errors and taking out time to review the project. I will make the changes immediately.

**Review by DeliceLydia — APPROVED** [review]
Hi @Ibizugbe, 

Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 
# Congratulations! 🎉 



### Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Comment by Ibizugbe** [comment]
Thanks, @DeliceLydia  for taking out time to review and approve the project

**Pull request merged**